crates/driver/src/domain/quote.rs

```diff
@@ -143,8 +147,8 @@ impl Order {
                 token: self.tokens.sell,
             },
             // Note that we intentionally do not use [`eth::U256::max_value()`]
-            // as an order with this would cause overflows with the Smart
-            // Contract, so buy orders requiring excessively large sell amounts
+            // as an order with this would cause overflows with the smart
+            // contract, so buy orders requiring excessively large sell amounts
```

Comment:
I think in theory, "Smart Contract" should be capitalized in the context of Ethereum Smart Contracts. At least this is how I thought the Ethereum foundation capitalized it.

Reply:
**Edit**: I stand corrected, it is definitely "smart contract" and I've been doing it wrong for a long time 🙈